### ISSUE TITLE
Chat bubbles display in plain English

### ISSUE THREAD

**Issue by msheff1** [mclemente/fvtt-module-polyglot#377]
### What happened?

Chat bubbles above tokens display in plain English. Should display in the Font used in the chat for the selected language.

### Reproduction Steps

1. Open foundry as GM or log in as player.
2. Send chat message with a Language selected.
3. Observe how chat bubble pop-up above token speaker displays in plain English.

Following relevant Core settings must be enabled:

- Enable Chat Bubbles
- Pan to Token Speaker
![image](https://github.com/mclemente/fvtt-module-polyglot/assets/100036854/622438fa-1630-4885-96a4-e96afa716fac)

Message displays above token like so:
![image](https://github.com/mclemente/fvtt-module-polyglot/assets/100036854/39ea10e2-05e7-44dd-9404-466ff0b0a959)
But displays in the chat window just fine:
![image](https://github.com/mclemente/fvtt-module-polyglot/assets/100036854/ca5ca9a8-00e4-4dc4-b3cc-09e1f4a3eb62)
Usage was normal:
![image](https://github.com/mclemente/fvtt-module-polyglot/assets/100036854/b647bf66-fdc5-48fd-abd2-eb8011194b01)

Behaviour occurs with GM selected token, or with Player selected (and owned) token.

Module config tabs are as follows:
![image](https://github.com/mclemente/fvtt-module-polyglot/assets/100036854/b74e50b2-5529-4ce2-a036-91b2ec1899dc)
![image](https://github.com/mclemente/fvtt-module-polyglot/assets/100036854/dbd19e01-03de-4165-9819-b1cbd6e15faf)
![image](https://github.com/mclemente/fvtt-module-polyglot/assets/100036854/a89440cb-fdb6-4e4b-a57e-dcbc889f3dcd)
![image](https://github.com/mclemente/fvtt-module-polyglot/assets/100036854/274328ab-b4ee-43e9-831f-aed0ab5990fc)

### Which game system, and which version, are you using?

D&D 5e 3.2.0

### Which Polyglot version are you using?

2.4.15

**Comment by mclemente** [comment]
To see the bubbles scrambled as a GM you need the "Scramble Messages for GM" setting enabled, though you won't see the bubbles scrambled if you're not controlling a token (having a token selected).

Try one of these:
- Test it as a player user.
- Toggle that setting on and have another user send a message while you are controlling a token that doesn't understand its language.

**Comment by msheff1** [comment]
Hi! Thanks for responding to this so fast.

I've been able to get this to work now, I think this is intended behaviour rather than a bug - sorry for my mistake!

Scramble Messages for GM was an important setting to have ON, and the GM is required to have a token that does not know the spoken language selected/controlled. The GM(token) sees the language in the polyglot script as intended. Also works with player 1 and player 2 in the same way - more importantly I think this also works when logged in as a player and with no token selected. Probably this is the Scramble Out of Character messages setting?

![image](https://github.com/mclemente/fvtt-module-polyglot/assets/100036854/51acfd30-b92c-427b-b188-b910c3dcccda)

Regardless, clearly this issue arose from my misunderstanding of the core functionality of the mod, so I thank you for your patience and apologise for any undue concern.
